metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -173,6 +176,112 @@ private void testDelayedConfigurationOperations(LocalLogManagerTestEnv logEnv,
         assertEquals(Collections.singletonMap(BROKER0, ApiError.NONE), future1.get());
     }
 
+    @Test
+    public void testFenceMultipleBrokers() throws Throwable {
```

Comment:
This test times out for me when run locally. I'm a little concerned that the dependence on real time here will make it flaky. I'm satisfied with the other tests we have in this patch. Would it be reasonable to push this test case to a follow-up so that we can iterate on it a little bit without blocking the patch?

Reply:
The timeout was because I accidentally deleted a line before pushing the commit. Will fix that. 
Will discuss pulling the test out offline with you.